externs/olx.js

```diff
@@ -5256,6 +5257,15 @@ olx.source.TileWMSOptions.prototype.crossOrigin;
 
 
 /**
+ * Additional "dimensions" for tile requests.  This is an object with properties
+ * named like the advertised dimensions (such as TIME).
+ * @type {Object|undefined}
+ * @api stable
```

Comment:
If this does make it through as a new option, I hope it can be kept experimental for at least one release.  That would allow us to make a more general solution later.